CryptoTechReminderSystem.Test/UseCase/GetLateDevelopersTests.cs

```diff
@@ -76,7 +83,13 @@ public void CanGetHarvestDevelopers()
         {
             var harvestGatewaySpy = new HarvestGatewaySpy();
             var slackGatewaySpy = new SlackGatewaySpy();
```

Comment:
These variables can be extracted into a `SetUp` class as this unit test suite does not have one.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/23/commits/d6455e552f5b3fc86caecb8c5d86964e3b5fc87f.